resources/views/users/print.blade.php

```diff
@@ -274,6 +275,9 @@ class="snipe-table table table-striped inventory"
                         <td>{{ ($accessory->manufacturer) ? $accessory->manufacturer->name : '' }} {{ $accessory->name }} {{ $accessory->model_number }}</td>
                         <td>{{ $accessory->category->name }}</td>
                         <td>{{ $accessory->pivot->created_at }}</td>
+                        <td><img style="width:auto;height:100px;" src="{{ asset('/') }}display-sig/{{ $accessory->assetlog->first()->accept_signature }}"></td>
```

Comment:
We probably want to check if a signature exists, otherwise we'll be printing broken images